src/pypromice/process/L0toL1.py

```diff
@@ -106,7 +106,9 @@ def toL1(L0, vars_df, flag_file=None, T_0=273.15, tilt_threshold=-100):
                                                     ds.attrs['pt_z_factor'], 
                                                     ds.attrs['pt_z_coef'], 
                                                     ds.attrs['pt_z_p_coef'])       
-            
+        ds['z_stake'] = _reformatArray(ds['z_stake'])                          # Reformat boom height    
+        ds['z_stake'] = ds['z_stake'] * ((ds['t_u'] + T_0)/T_0)**0.5           # Adjust sonic ranger readings for sensitivity to air temperature
+        
     elif ds.attrs['number_of_booms']==2:                                       # 2-boom processing
         ds['z_boom_l'] = _reformatArray(ds['z_boom_l'])                        # Reformat boom height    
         ds['z_boom_l'] = ds['z_boom_l'] * ((ds['t_l'] + T_0)/T_0)**0.5         # Adjust sonic ranger readings for sensitivity to air temperature
```

Comment:
Why isn't `z_boom_u` correction done here?

Reply:
And, just generally, I would recommend more descriptive branch names than `bug-fix-bav`. Something like `correct-z-stake-sonic` would be better. And also good practice is to open a GH issue for this, which this PR then addresses and closes. Not necessary here, just a recommendation for the future.

Reply:
>Why isn't z_boom_u correction done here?

It is done earlier as all station should have `z_boom_u` and `t_u`:
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/08cd0c86a474f73ef9472eb9b1863c871048a2db/src/pypromice/process/L0toL1.py#L59-L60

Noted for the branch naming and issue. Will do next time!